google/data_source_google_active_folder.go

```diff
@@ -14,7 +14,8 @@ func dataSourceGoogleActiveFolder() *schema.Resource {
 		Schema: map[string]*schema.Schema{
 			"parent": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
```

Comment:
Can you update https://github.com/hashicorp/terraform-provider-google/blob/master/website/docs/d/active_folder.html.markdown as well?

Reply:
Done~